### PR TITLE
Feature: Thesauri improvement

### PR DESCRIPTION
**Pull request by Ali-GeoFZ**
This PR updates the Thesauri Keywords form group layout and its tests to reflect the new, non-accordion structure.

## Changes

- Removed the accordions from the Thesauri Keywords form group.
- Removed the tree connector lines that were shown when the tree was expanded.
- Updated the tests so they no longer target the accordion, but instead check the individual fields and the form group.

## Notes for Reviewer

- Please run the updated tests carefully and check whether I have missed any gaps or edge cases in the test coverage.

## Checklist
- [x] My code follows the style guide.
- [x] I have self-reviewed my code.
- [x] I added comments for hard-to-understand code.
- [x] If applicable, PHP code is documented using PHPDoc.
- [x] If applicable, JavaScript code is documented using JSDoc.
- [x] If needed, the ELMO Guide has been updated.
- [x] If needed, the README has been updated.
- [x] If needed, the API documentation has been updated.
- [x] If a new feature was added or a bug fixed, the changelog has been updated.
- [x] My changes do not create new warnings in the test browser console.
- [x] I have added unit tests that cover my code.
- [x] All new and existing unit tests pass locally.
- [x] All new and existing automated unit tests pass in the pull request.
- [x] If applicable, Playwright tests have been updated and new tests added.
- [x] All new and existing automated Playwright tests pass in the pull request.
- [x] I ensured the changes meet accessibility guidelines.

## Known Issues
None.

### CODE REVIEW

File: tests/js/thesauri.test.js [hunk 1]
```diff
@@ -438,9 +431,7 @@ describe('thesauri.js — showLoadingSpinner / hideLoadingSpinner / loadThesauru
     document.body.innerHTML = `
       <div id="thesaurusKeywordsFormGroup" class="card mb-2" style="display: none;">
         <div class="card-header"><b data-translate="keywords.thesaurus.name"></b></div>
-        <div id="thesaurusKeywordsGroup">
-          <div class="accordion p-2" id="accordionThesauri"></div>
-        </div>
+        <div id="thesaurusKeywordsGroup"></div>
```
Comment (Daetha) [inline tests/js/thesauri.test.js:434]: this looks good. a div to receive the generated HTML

File: tests/playwright/formgroups/thesauri-keywords.spec.ts
Comment (Daetha) [inline tests/playwright/formgroups/thesauri-keywords.spec.ts:289]: this code is outdated. there is no more modal

File: tests/playwright/formgroups/thesauri-keywords.spec.ts [hunk 2]
```diff
@@ -300,13 +289,13 @@ test.describe('Thesauri Keywords Form Group', () => {
...
     // Click on the Tagify input to trigger focus-based lazy loading
-    const tagifyInput = page.locator('#collapse-science_keywords .tagify__input');
+    const tagifyInput = page.locator('#input-sciencekeyword').locator('..').locator('.tagify__input');
     await tagifyInput.click();
 
     // Wait for the API call to complete and whitelist to be populated
     await page.waitForFunction(() => {
       const input = document.getElementById('input-sciencekeyword') as any;
-      return input?._tagify?.settings?.whitelist?.length > 0;
+      return input?.tagify?.settings?.whitelist?.length > 0 || input?._tagify?.settings?.whitelist?.length > 0;
```
Comment (Daetha) [inline tests/playwright/formgroups/thesauri-keywords.spec.ts]: this basically says: wherever the whitelist lives: in tagify or in _tagify, it has to be not-empty
makes me worry.. why cant we tell for sure where the whitelist is? Can you please investigate this issue?

File: tests/playwright/formgroups/thesauri-keywords.spec.ts [hunk 3]
```diff
@@ -235,61 +233,52 @@ test.describe('Thesauri Keywords Form Group', () => {
...
 
     for (const config of sectionConfigs) {
-      const button = page.locator(`button[data-bs-target="#${config.collapseId}"]`);
-      await expect(button).toHaveAttribute('aria-controls', config.collapseId);
-      await expect(button).toHaveAttribute('aria-expanded', config.expanded);
-      await expect(button).toHaveText(config.name);
+      const input = page.locator(config.inputId);
+      const item = input.locator('xpath=ancestor::div[contains(@class,"thesaurus-input-item")]');
+
+      await expect(item).toBeVisible();
```
Comment (Daetha) [inline tests/playwright/formgroups/thesauri-keywords.spec.ts:272]: analogous to deleted aria-expanded check (no longer needed, there is no modal to expand) you can check if the input is not `aria-hidden`

File: tests/playwright/formgroups/thesauri-keywords.spec.ts [hunk 4]
```diff
@@ -427,16 +417,14 @@ test.describe('Thesauri Keywords Form Group', () => {
     await platformsModal.locator('.modal-footer button.btn-primary').click();
     await expect(platformsModal).toBeHidden();
 
-    const platformTags = page.locator('#collapse-platforms .tagify__tag');
+    const platformTags = page.locator('#input-platforms').locator('..').locator('.tagify__tag');
```
Comment (Daetha) [inline tests/playwright/formgroups/thesauri-keywords.spec.ts:421]: this is good in terms of readability

**Review by Daetha — CHANGES_REQUESTED** [review]
Dear Ali, thank you for your work. I needed less clicks to access the thesaurus entry, which feels good. 

What is more, save and Upload button does work, but the clear button does not work. 

Removing the accodions also means removing the titles in them. I feel like the text: Open thesaurus to choose keywords or start typing... is repeated 5 times without a reason. But there is no indication, which KW vocabulary is populated by this field: is it GCMD Platforms or Science or GEMET? They all look the same. 

This is why I recommend to add the names of thesauri into the input fields so that users dont get confused which vocabulary is where. 

I hope my comments are clear and polite ☺️,
Best regards,
Sasha

File: tests/playwright/formgroups/thesauri-keywords.spec.ts
Comment (Ali-GeoFZ) [reply tests/playwright/formgroups/thesauri-keywords.spec.ts:289]: The accordion was removed, but the thesaurus modals (e.g. #modal-sciencekeyword) are still part of the current implementation and used in the tests, so this code is not outdated.

File: tests/playwright/formgroups/thesauri-keywords.spec.ts (hunk 2 above)
Comment (Ali-GeoFZ) [reply tests/playwright/formgroups/thesauri-keywords.spec.ts]: You’re right

File: tests/playwright/formgroups/thesauri-keywords.spec.ts (hunk 3 above)
Comment (Ali-GeoFZ) [reply tests/playwright/formgroups/thesauri-keywords.spec.ts:272]: You're absolutely right, I've added it now

**Comment by Ali-GeoFZ** [comment]
> Dear Ali, thank you for your work. I needed less clicks to access the thesaurus entry, which feels good.
> 
> What is more, save and Upload button does work, but the clear button does not work.
> 
> Removing the accodions also means removing the titles in them. I feel like the text: Open thesaurus to choose keywords or start typing... is repeated 5 times without a reason. But there is no indication, which KW vocabulary is populated by this field: is it GCMD Platforms or Science or GEMET? They all look the same.
> 
> This is why I recommend to add the names of thesauri into the input fields so that users dont get confused which vocabulary is where.
> 
> I hope my comments are clear and polite ☺️, Best regards, Sasha

Dear Sasha,

thank you very much for your detailed review and the kind notes.
I’ve implemented all of your suggestions and would be happy if you could take another look at it

Best regards
Ali

**Review by Daetha — APPROVED** [review]
Dear Ali, the tests are completed successfully: 

✓ autocomplete when input is focused without opening modal first
✓ input contains valid JSON for backend after loading XML keywords
✓ processKeywords loads tags with all metadata fields including language
thesauri-keywords.spec.ts

✓ renders accessible thesaurus input sections and controls
✓ keyword selections between tree, summary list, and Tagify input
✓ searching, keyboard access, and persistence across thesauri modals
✓ updates Tagify placeholders when switching languages


The clear works as well. 

Known issue: #1102
